extensions/graphviz_support/graphviz_support/tests/demo_graph/amplifier.py

```diff
@@ -4,9 +4,16 @@
 import labgraph as lg
 import numpy as np
 import time
+import random
 from .random_message import RandomMessage
+from datetime import datetime
+from time import perf_counter
+from collections import deque 
 
-
+data = deque()
```

Comment:
In the future, deque can be replaced by a more performant buffer data structure.